Tools/LibEnsemble/sim/inputs

```diff
@@ -156,7 +156,7 @@ beam.z_cut = 2.
 beam.x_m = 0.
 beam.y_m = 0.
 beam.z_m = -98.e-6
-beam.npart = 10000
+beam.npart = 100000
```

Comment:
Since I am a bit off topic, I was wondering.
Do you need the increase in particle number or to do to the analysis we used to do when we did convergence scans before?
I mean the type of selection of particles that form the actual "beam" falling within RMS distances from centroid.
What do you think?

Reply:
The statistics was a bit too low for a convergence tests.